pkg/assume/assume.go

```diff
@@ -99,6 +101,48 @@ func AssumeCommand(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+
+	if assumeFlags.Bool("url") {
+		//dont want to open the browser just return the link
+		labels := browsers.RoleLabels{Profile: profile.Name}
+
+		var creds aws.Credentials
+
+		creds, err = profile.AssumeConsole(c.Context)
+		if err != nil {
+			return err
+		}
+
+		service := assumeFlags.String("service")
+		if assumeFlags.String("region") != "" {
+			region = assumeFlags.String("region")
+		}
+
+		labels.Region = region
+		labels.Service = service
+		err, url := browsers.MakeUrl(browsers.SessionFromCredentials(creds), labels, service, region)
+		if err != nil {
+			return err
+		}
+
+		cfg, _ := config.Load()
+		if cfg == nil {
+			return browser.OpenURL(url)
+		}
+		if cfg.DefaultBrowser == browsers.FirefoxKey {
+			url = browsers.MakeFirefoxContainerURL(url, labels)
+			if err != nil {
+				return err
+			}
+			browsers.ManuallyOpenURL(url)
```

Comment:
can we replace this with the stdout print, I think we can get away with not printing a friendly message here because you get this result by specifying the -url flag, so you expect to get a url
```suggestion
			fmt.Print(url)
```